src/pages/public_registration/action_accordions/DeletePublication.tsx

```diff
@@ -0,0 +1,154 @@
+}
+
+interface DeletePublicationProps {
+  registration: Registration;
+}
+
+export const DeletePublication = ({ registration }: DeletePublicationProps) => {
+  const [showDeleteButton, setShowDeleteButton] = useState(false);
+  const [showDeleteModal, setShowDeleteModal] = useState(false);
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const [selectedDuplicate, setSelectedDuplicate] = useState<Registration | null>(null);
+  const history = useHistory();
+
+  const deleteValidationSchema = Yup.object().shape({
+    deleteMessage: Yup.string()
+      .min(3, t('feedback.validation.must_be_bigger_than', { field: i18n.t('common.justification'), limit: 3 }))
+      .required(t('feedback.validation.is_required', { field: i18n.t('common.justification') })),
+  });
+
+  const unpublishRegistrationMutation = useMutation({
+    mutationFn: (values: DeleteForm) => {
+      if (selectedDuplicate) {
+        return unpublishRegistration(registration.identifier, {
+          type: 'UnpublishPublicationRequest',
+          duplicateOf: selectedDuplicate.id,
+          comment: values.deleteMessage,
+        });
+      } else {
+        return unpublishRegistration(registration.identifier, {
+          type: 'UnpublishPublicationRequest',
+          comment: values.deleteMessage,
+        });
+      }
+    },
+    onSuccess: () => {
+      setShowDeleteModal(false);
+      history.push(`${getRegistrationLandingPagePath(registration.identifier)}?shouldNotRedirect`);
+    },
+    onError: () => {
+      dispatch(setNotification({ message: t('feedback.error.update_registration'), variant: 'error' }));
+    },
+  });
+
+  return (
+    <>
+      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem', mt: '1rem' }}>
+        <Divider />
+        {!showDeleteButton && (
+          <IconButton
+            sx={{ alignSelf: 'center' }}
+            title={t('common.show_more_options')}
+            onClick={() => setShowDeleteButton(true)}>
+            <MoreHorizIcon />
+          </IconButton>
+        )}
+        {showDeleteButton && (
+          <Button
+            sx={{ alignSelf: 'center' }}
+            data-testid={dataTestId.unpublishActions.openUnpublishModalButton}
+            variant="outlined"
+            onClick={() => setShowDeleteModal(true)}>
+            {t('common.delete')}
+          </Button>
+        )}
+      </Box>
+      <Modal
+        dataTestId="delete-registration-modal"
+        headingText={t('registration.delete_registration')}
+        open={showDeleteModal}
+        onClose={() => setShowDeleteModal(false)}>
+        <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+          <RequiredDescription />
+          <Typography variant="h3">{t('unpublish_actions.unpublish_registration')}</Typography>
+          <Typography>{t('unpublish_actions.unpublish_registration_detail_1')}</Typography>
+          <Typography>{t('unpublish_actions.unpublish_registration_detail_2')}</Typography>
```

Comment:
Vi er i ferd med å ta i bruk en ny praksis her, hvor oversetter selv kan bestemme over avsnitt, da det ikke nødvendigvis skal være likt for alle språk. Må ikke fikse det her også, men kan se på `<Trans>` helt nederst [her](https://github.com/BIBSYSDEV/NVA-Frontend/pull/5163/files#diff-1833a6f6a362478e4528448fa34e680c6a9c3e049154e36baafb855206c0531c) for hvordan det kan gjøres for læringens del :)